examples/multi_region/README.md

```diff
@@ -0,0 +1,29 @@
+# Simple Example
+
+This example illustrates how to use the `sdwan` module.
```

Comment:
I like the examples included to the module 👍

Reply:
yes, in fact the example is what gets tested.